script.js

```diff
@@ -54,6 +54,12 @@ let projectData = [
     projectImage: "assets/GIFs/bouncingBalls.gif",
     projectUrl: "public/BouncingBalls.html",
   },
+  {
+    projectName: "To-do list",
+    projectImage: "assets/GIFs/todo.gif",
+    projectUrl: "public/Todo.html",
```

Comment:
Fix this as well, project is not loading properly.
Make it todo.html

Reply:
sure